.github/workflows/golangci-lint.yml

```diff
@@ -12,10 +12,12 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v4
+      - uses: actions/checkout@v4
+      - id: goversion
+        run: |
+          cat Dockerfile | awk '/^FROM golang:.* AS builder$/ {v=$2;split(v,a,":");split(a[2],b,"-")}; END {printf("version=%s", b[1])}' >> $GITHUB_OUTPUT
+      - uses: actions/setup-go@v5
```

Comment:
While you're here, can you configure Dependabot to handle Actions too please? I was wondering why we didn't get this version bump.

Reply:
Will do :)